policy/modules/system/init.te

```diff
@@ -519,6 +523,9 @@ ifdef(`init_systemd',`
 	# for systemd to read udev status
 	udev_read_runtime_files(init_t)
 
+	udev_relabel_rules_dirs(init_t)
+	udev_relabel_rules_files(init_t)
+
```

Comment:
I dont think systemd relabels /etc let alone /etc/udev?

Reply:
Likewise, this is probably related to using the `udev-rules` dracut module.

Reply:
Hm. Even after removing that module I still see this denial:
```
audit: type=1400 audit(1616864322.185:5): avc:  denied  { relabelto } for  pid=1 comm="systemd" name="rules.d" dev="tmpfs" ino=9 scontext=system_u:system_r:init_t:s0 tcontext=system_u:object_r:udev_rules_t:s0 tclass=dir permissive=1
```